### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by kojix2**
Hi!

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,20 +6,23 @@ on:
...
-      - name: build
-        run: shards build --production
+      - name: install dependencies
+        run: |
+          sudo apt-get install libdbus-1-dev libnotify-dev
+          shards install
+      - name : build
+        run: crystal build src/notify.cr
+      # - name: Run specs
+      #   run: crystal spec
```
Comment (woodruffw) [inline .github/workflows/ci.yml:28]: Why is this commented out? Lack of DBus on CI?

**Review by woodruffw — APPROVED** [review]
LGTM overall, one question about the changes.

Thanks for contributing these fixes!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kojix2) [reply .github/workflows/ci.yml:28]: I searched GitHub and checked several repositories that call notify, but could not find any that had successfully set up GitHub Actions. I found that even the Rust library [notify-rust](https://github.com/hoodie/notify-rust/blob/main/.github/workflows/ci.yml) has the test commented out. So I guessed that there seemed to be no easy way to do this.

Comment (woodruffw) [reply .github/workflows/ci.yml:28]: Got it. Yeah, this is probably going to be pointlessly annoying to test.

**Comment by woodruffw** [comment]
Thanks @kojix2!

**Pull request merged**